promscale/visualize-data/grafana.md

```diff
@@ -0,0 +1,80 @@
+
+To connect Grafana with Promscale, install and run Grafana version 5.3 or later. For information about installing Grafana, see [Grafana installation docs][grafana-install].
+
+This section shows you how to connect Promscale to Prometheus, Jaeger, and PostgreSQL data sources in [Grafana][grafana-homepage].
+
+<procedure>
+
+## Configuring Promscale as Prometheus data source
+
+1.  Navigate to `Configuration` → `Data sources`.
+    The data sources page opens showing a list of previously configured data sources for the Grafana instance.
+1.  Click `Add data source`to see a list of all supported data sources.
+1.  Type `Prometheus` in the search field and click `Select`.
+1.  Configure the data source:
+    *   In the `Name`field, type `Promscale-metrics`.
+    *   In the `URL` field, type `http://<PROMSCALE-IP-ADDR>:9201`, where `<PROMSCALE-IP-ADDR>` is the IP address of the Promscale instance.
+    *   Use the default values for all other settings.
+
+</procedure>
+
+When you have configured Promscale as a Prometheus data source in Grafana, you can create panels that are populated with data using PromQL.
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/images/misc/getting-started-with-promscale-grafana-dashboard.png" alt="Sample output for PromQl query"/>
+
+<procedure>
+
+## Configuring Promscale as Jaeger data source
+
+Grafana ships with built-in support for Jaeger, which provides open source, end-to-end distributed tracing.
+
+1.  Navigate to `Configuration` → `Data sources`.
+    The data sources page opens showing a list of previously configured data sources for the Grafana instance.
+1.  Click `Add data source`to see a list of all supported data sources.
+1.  Type `Jaeger` in the search field and click `Select`.
+1.  Configure the data source settings:
+    *   In the `Name`field, type `Promscale-traces`.
+    *   In the `URL`field, type `http://<PROMSCALE-IP-ADDR>:9201`, where `<PROMSCALE-IP-ADDR>`is the IP address of the Promscale instance.
+    *   Use the default values for all other settings.
+
+</procedure>
+
+You can now filter and view traces stored in Promscale using Grafana. To visualize your traces, click the **Explore** button to see the traces filtering panel.
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/images/misc/grafana-jaeger-query-results.png" alt="Sample output for Jaeger query filter in Grafana"/>
+
+<procedure>
+
+## Configuring Promscale as PostgreSQL data source
+
+To configure Promscale as PostgreSQL data source you need details such as host, port, database, user, and password. These are available in the service URL that you copied when you created the Timescale cloud service. The format of that URL is `postgresql://[user[:password]@][host][:port][/dbname][?param1=value1&...]`
```

Comment:
@Rajakavitha1 here the context isn't **Timescale Cloud Service** the reference blog post that we published is focused on Timescale cloud service being configured as the data source in Grafana. I would change the phrase from
```
These are available in the service URL that you copied when you created the Timescale cloud service. The format of that URL is
```

to


```
These details should be available in the PostgreSQL service URL. The format of that URL should be
```